manifests/nodejs.yml

```diff
@@ -1304,11 +1316,11 @@ tests/:
       Test_Parametric_DDSpan_Set_Resource: incomplete_test_app (set_resource endpoint is not implemented)
       Test_Parametric_DDSpan_Start: bug (APMAPI-778)  # The resource name of the child span is overidden by the parent span.
       Test_Parametric_DDTrace_Baggage: incomplete_test_app (baggage endpoints are not implemented)
-      Test_Parametric_DDTrace_Crash: incomplete_test_app (crash endpoint is not implemented)
+      Test_Parametric_DDTrace_Crash: v5.67.0
```

Comment:
In the Node.js tracer we use a different way to refer to the versions rather than the actual version number, so this technically should be `*ref_5_67_0` instead of `v5.67.0`. But I'm sure that will complicate the automated script (also since `ref_5_67_0` currently doesn't exists above, so it should be added in the same PR).

There's no different between `ref_5_67_0` and `v5.67.0` currently because we currently only have one active release line, but once we get two again (which will happen very soon), this will become an issue.

Reply:
I would say it might be a silver lining actually. Consider this: the bot cannot create the alias easily, thus it's setting a raw vX.X.X version. But this version is also actually not guaranteed to be correct, since it's only taking the latest version, not the actual version where the feature was released. This bot is a failsafe, not something we should rely on. The teams should activate system tests with the verified released numbers for all branches etc. But since the bot cannot do that, we will be able to quickly look at the manifest, and immediately spot places where the bot did a "best-effort" job, that we should either double check and correct, by just looking for those vX.X.X instead of our usual refs. Does that make sense? it's a bit difficult to explain